src/main/java/uk/gov/companieshouse/pscstatementdataapi/services/RetrievePscStatementService.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.pscstatementdataapi.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.psc.Statement;
+import uk.gov.companieshouse.pscstatementdataapi.model.PscStatementDao;
+import uk.gov.companieshouse.pscstatementdataapi.repository.PscStatementRepository;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.Optional;
+@Service
+public class RetrievePscStatementService {
```

Comment:
Could this service be renamed to PscStatementService? It will do more than just retrieve PSCs as it will be responsible for saves and deletes as well